src/js/app.js

```diff
@@ -11,34 +12,48 @@ const app = {
 
     thisApp.pages = document.querySelector(select.containerOf.pages).children;
     thisApp.navLinks = document.querySelectorAll(select.nav.links);
-
-    const idFromHash = window.location.hash.replace('#/', '');
-    console.log(idFromHash);
-
+    // const idFromHash = window.location.hash.replace('#/', '');
     let pageMatchingHash = thisApp.pages[0].id;
 
-    for(const page of thisApp.pages){
-      if(page.id == idFromHash){
-        pageMatchingHash = page.id;
-        break;
-      }
-    }
+    // Commented to make main page always active on reload.
+    // for(const page of thisApp.pages){
+    //   if(page.id == idFromHash){
+    //     pageMatchingHash = page.id;
+    //     break;
+    //   }
+    // }
 
     thisApp.activatePage(pageMatchingHash);
 
-    for(let link of thisApp.navLinks){
-      link.addEventListener('click', function(event){
-        const clickedElement = this;
-        event.preventDefault();
+    window.location.hash = '#/' + pageMatchingHash;
+
+    thisApp.changePageListener(select.nav.linksWrapper);
 
-        const id = clickedElement.getAttribute('href').replace('#', '');
+  },
 
-        thisApp.activatePage(id);
+  changePageListener(linksWrapper){
```

Comment:
dlaczego ta funkcja nazywa się `change...`? Co ona zmienia?

Reply:
Najbliższa lepsza nazwa jaka przychodzi mi do głowy to `addListenerForPageChange`